cns/restserver/util.go

```diff
@@ -177,7 +176,7 @@ func (service *HTTPRestService) saveNetworkContainerGoalState(req cns.CreateNetw
 				service.state.ContainerIDByOrchestratorContext = make(map[string]string)
 			}
 
-			service.state.ContainerIDByOrchestratorContext[podInfo.PodName+podInfo.PodNamespace] = req.NetworkContainerid
```

Comment:
>[podInfo.PodName+podInfo.PodNamespace [](http://example.com/codeflow?start=49&length=37)

This will fail the ACI scenario: previously Key is PodName+PodNamespae (Pod1podns).....and now the Key is PodName:PodNameSpace......

Please note this map is stored in the statefile, so ACI update will fail